demos/tensorflow-image-tagging/README.md

```diff
@@ -0,0 +1,36 @@
+# Image Search and Tagging
+
+The demo apps are HTTP servers designed specifically for exercising demo scenarios in Waiter.
+This demo application performs image search and then tags the images using the [inception-v3 model built in tensorflow](https://www.tensorflow.org/tutorials/image_recognition).
+
+The demo highlights the following features of Waiter:
+
+1. Registering image search and image tagging as DNS record tokens in Waiter.
+
+1. Running different services (image search and image tagging) on Waiter.
+
+1. Services with different concurrency level support.
+
+1. Invoking different services from the same client (i.e. the demo page).
+
+1. [Scaling of services](../../waiter/docs/autoscaling.md) (image tagging).
+
+
+## Running the demo
+
```

Comment:
Add prerequisites:

1. Java 8
2. Leiningen
3. Docker

Can we prebuild a Waiter image for them to use, so they don't need 1 and 2?